### PR TITLE
docs(input): add showCount version

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/input/index.en-US.md](https://github.com/ant-design/ant-design/blob/docs/components/input/index.en-US.md)
[View rendered components/input/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/docs/components/input/index.zh-CN.md)

### CODE REVIEW

File: components/input/index.zh-CN.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Input 的其他属性和 React 自带的 [input](https://facebook.github.io/reac
 | allowClear | 可以点击清除图标删除内容 | boolean | false |  |
 | onResize | resize 回调 | function({ width, height }) | - |  |
 | bordered | 是否有边框 | boolean | true | 4.5.0 |
-| showCount | 是否展示字数 | boolean | false |  |
-| maxLength | 内容最大长度 | number | - |  |
+| showCount | 是否展示字数 | boolean | false | 4.7.0 |
```
Comment (xrkffgg) [inline components/input/index.zh-CN.md:53]: Emmm  排下序呗

Comment (hengkx) [reply components/input/index.zh-CN.md:53]: 现在都是 按字母排序了吗？

Comment (xrkffgg) [reply components/input/index.zh-CN.md:53]: 嗯。https://github.com/ant-design/ant-design/wiki/API-Naming-rules#promise

有空了，我就在改。碰到别人在改，我就提醒下 😂😂😂

Comment (hengkx) [reply components/input/index.zh-CN.md:53]: 这个 没问题了 合了吧，我写个 脚本 批量 处理下

**Pull request merged**